EXOSIMS/SurveySimulation/multiSS.py

```diff
@@ -0,0 +1,904 @@
+                        else:
+                            tAbs = (
+                                TK.missionStart + TK.missionLife
+                            )  # advance to end of mission
+                        tmpcurrentTimeNorm = TK.currentTimeNorm.copy()
+
+                        # Advance Time to this time OR start of next
+                        # OB following this time
+                        if int(np.ceil(TK.currentTimeNorm.copy().value)) >= 1790:
+                            tAbs = TK.currentTimeAbs.copy() + 20
+                        _ = TK.advanceToAbsTime(tAbs, self.defaultAddExoplanetObsTime)
+                        self.vprint(
+                            (
+                                "No Observable Targets a currentTimeNorm= {:.2f}. "
+                                "Advanced To {:.2f}"
+                            ).format(
+                                tmpcurrentTimeNorm.to("day"),
+                                TK.currentTimeNorm.to("day"),
+                            )
+                        )
+        else:  # TK.mission_is_over()
+            dtsim = (time.time() - t0) * u.s
+            log_end = (
+                "Mission complete: no more time available.\n"
+                + "Simulation duration: %s.\n" % dtsim.astype("int")
+                + "Results stored in SurveySimulation.DRM (Design Reference Mission)."
+            )
+            self.logger.info(log_end)
+            self.vprint(log_end)
+
+    def reset_sim(self, genNewPlanets=False, rewindPlanets=True, seed=None):
+        """Performs a full reset of the current simulation.
+
+        This will reinitialize the TimeKeeping, Observatory, and SurveySimulation
+        objects with their own outspecs.
+
+        Args:
+            genNewPlanets (bool):
+                Generate all new planets based on the original input specification.
+                If False, then the original planets will remain. Setting to True forces
+                ``rewindPlanets`` to be True as well. Defaults True.
+            rewindPlanets (bool):
+                Reset the current set of planets to their original orbital phases.
+                If both genNewPlanets and rewindPlanet  are False, the original planets
+                will be retained and will not be rewound to their initial starting
+                locations (i.e., all systems will remain at the times they were at the
+                end of the last run, thereby effectively randomizing planet phases.
+                Defaults True.
+            seed (int, optional):
+                Random seed to use for all random number generation. If None (default)
+                a new random seed will be generated when re-initializing the
+                SurveySimulation.
+
+        """
+        SU = self.SimulatedUniverse
+        TK = self.TimeKeeping
+        TL = self.TargetList
+        # uncommment the next line to run simulation on the same
```

Comment:
you cannot introduce functionality through source code changes.  That's not allowed.  If you want something to be toggleable, make it a user input and add appropriate logic.